fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/GroupByTest.java

```diff
@@ -271,7 +276,7 @@ void testBitMapWithStreamAggregation() {
                                         indexPlan()
 
                                 )).where(aggregations(recordConstructorValue(exactly(bitmapAggregationValue(anyValue()))))
-                                .and(groupings(ValueMatchers.fieldValueWithFieldNames("select_grouping_cols"))))));
+                                .and(groupings(ValueMatchers.anyValue())))));
```

Comment:
you can still verify that the grouping columns are `str_value_indexed`.

Reply:
This is not resolved properly.

Reply:
This is not resolved.

Reply:
I changed it to `.and(groupings(recordConstructorValue(exactly(anyValue(), anyValue())))))));` but replacing `anyValue()` with `ValueMatchers.findFieldValueWithNames("str_value_indexed")` doesn't work as I expected it should be.

Reply:
Fixed.